test/webrtc_test.rb

```diff
@@ -142,53 +163,71 @@ def configure_and_start()
 
         assert_equal state_task, state[0]
         assert_equal state_task, state[1]
```

Comment:
The websocket should be valid after a stop. Only a cleanup should kill it. (`state_task.web_socket = :NoWebSocket` above)

Reply:
Indeed...I will work on that.
Always after close datachannel and peerconnection the websocket OnClosed callback is called, so I will put a promise on the stopHook, as well, to properly wait for that close, and in the cleanup I will reset the pointer.